src/flows/templates/index.ts

```diff
@@ -1,6 +1,7 @@
 export interface Template {
   type: string // a unique string that identifies a visualization
   init: (...args: string[]) => any // returns a flow state that will pass the hydrate function in src/flows/context/flow.list.tsx
+  callback?: (...args: string[]) => any // optional callback func to return the newly created notebook id
```

Comment:
dont we do this some other way here?

Reply:
I couldn't find any other mechanism to get the notebook ID of the created notebook from a template 🤷 If one exists, I'm all ears